can't resolve crypt module

Latest version 9.16.1 results in the following error when used in an angular build while setting version back to 9.16.0 runs a successful build.

ERROR in ./node_modules/auth0-js/dist/auth0.min.esm.js
Module not found: Error: Can't resolve 'crypto' in './node_modules/auth0-js/dist'

--- a/package.json
+++ b/package.json
@@ -53,7 +53,7 @@
   },
   "dependencies": {
     "base64-js": "^1.3.0",
-    "idtoken-verifier": "^2.1.1",
+    "idtoken-verifier": "^2.1.2",
     "js-cookie": "^2.2.0",
     "qs": "^6.7.0",
     "superagent": "^5.3.1",

